### ISSUE TITLE
exports defaults & steal.import is changing behavior after building

### ISSUE THREAD

**Issue by bkrrr** [stealjs/steal#611]
Hello

assuming i have a module (example.js):
`export default function(){
 }`

and loading somewhere else:
```
steal.import('./example').then(function(promise) {
}
```

before building with steal-tools i can call
` var test  = new promise.default()`

after building i have to call:
`  var test  = new promise()`
_________
For anybody who is coming across this post that's the walkaround:
```
var test;
if(promise instanceof Function){
				 test  project = new promise();
			}else{
				test  project = new promise.default();
			}
```

**Comment by nlundquist** [comment]
+1 i've experienced this also

**Comment by matthewp** [comment]
This is fixed in 1.0 (Woot!)
